tidb-cloud-in-public-preview.md

```diff
@@ -0,0 +1,49 @@
+---
+title: TiDB Cloud Now in Public Preview
+author: [PingCAP]
+date: 2021-05-10
+summary: We are excited to announce the public preview of TiDB Cloud, the fully-managed TiDB service by PingCAP. With TiDB Cloud, customers can now easily use TiDB on Amazon Web Services and Google Cloud to quickly build modern, mission critical applications.
+tags: ['Announcement', 'TiDB Cloud']
+categories: ['Product']
+image: /images/blog/tidb-cloud-in-public-preview.jpg
+press_release: true
+---
+
+![TiDB Cloud in Public Preview](media/tidb-cloud-in-public-preview.jpg)
+
+We are excited to announce the public preview of TiDB Cloud, the fully-managed TiDB service by PingCAP. With TiDB Cloud, customers can now easily use TiDB on Amazon Web Services and Google Cloud to quickly build modern, mission critical applications.
+
+## Get the full power of TiDB
+
+TiDB is the leading open-source, MySQL compatible, distributed NewSQL database in the industry that supports hybrid transactional and analytical processing (HTAP), which allows business to run real-time analytical queries. It also features horizontal scalability, strong consistency, and high availability.
+
+*   **More control over your scaling**: Scale your performance (TiDB) and storage (TiKV) nodes separately 
+*   **High availability and reliability**: Ensure business continuity for your mission critical applications with our ACID-compliant cluster deployment
+*   **Scalable Multi-Master: **No more bottlenecks - scale without restrictions with read and write capability on every node
+*   **High Concurrency**: Have thousands of users query your data at the same time without sacrificing performance 
+*   **Simplified Database Architecture**: Eliminate complex data migration between transactional and analytical workloads by storing data in a single database  
+*   **Real Time Operational Analytics**: Run real-time analytical queries with our hybrid transactional-analytical processing (HTAP) capabilities 
+
+TiDB has been growing for the past 6 years, and is now deployed in production by over 1500 companies worldwide. Open source since day one, TiDB has donated two projects to the Cloud Native Computing Foundation (CNCF) and currently has over 400 contributors to its [Github repository](https://github.com/pingcap/tidb).
```

Comment:
```suggestion
TiDB has been growing for the past 6 years, and is now deployed in production by over 1500 companies worldwide. Open source since day one, TiDB has donated two projects to the Cloud Native Computing Foundation (CNCF) and currently has over 400 contributors to its [GitHub repository](https://github.com/pingcap/tidb).
```